app/models/micropost.rb

```diff
@@ -0,0 +1,37 @@
+class Micropost < ApplicationRecord
+  CONTENT_LIMIT = 140
+  IMAGE_RESIZE_LIMIT = [500, 500].freeze
+  IMAGE_SIZE_LIMIT = 15 # MBs
+  IMAGE_PERMIT = %w(image/jpeg image/gif image/png).freeze
+
+  MICROPOST_PERMIT = %i(
+    content
+    image
+  ).freeze
+
+  has_one_attached :image do |attachable|
+    attachable.variant :display, resize_to_limit: IMAGE_RESIZE_LIMIT
+  end
+
+  belongs_to :user
+
+  scope :recent_posts, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :recent_posts, -> {order(created_at: :desc)}
```
Thêm space trước block